cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1673,7 +1672,7 @@
 
   <property>
     <name>messaging.max.instances</name>
-    <value>1</value>
+    <value>${master.service.max.instances}</value>
     <description>
       Maximum number of instances for the messaging service
```

Comment:
We should update the description, saying it does not increase scalability but only used for HA.